File: Project-2/Message.py
class Message:
    numThread=""
    def enterMuseum(self):
        print (self.numThread , "enter Museum")

    def waitingForTaxi(self):
        print(self.numThread , "waiting for taxi")

    def enterTaxi(self,numTaxi):
        print(self.numThread , "enter to taxi" , numTaxi)

    def inTaxi(self,numTaxi):
        print("\t" , numTaxi , "started")

    def exitTaxi(self,numTaxi):
        print("\t" , numTaxi , "stoped")

File: Project-2/Message.java
package message;

public class Message {
    String nameThread;
    public void enterMuseum(){
        System.out.println(nameThread + " enter museum");

    }
    public void waitingForTaxi(){
        System.out.println(nameThread + " waiting for taxi");
    }
    public void enterTaxi(int numTaxi){
        System.out.println(nameThread + " enter to taxi " + numTaxi);
    }
    public void inTaxi(int numTaxi) {
        System.out.println("\t taxi " + numTaxi + " start");
    }
    public void exitTaxi(int numTaxi){
        System.out.println("\t taxi " + numTaxi + " stop");
    }
}
